lib/rules/value-no-vendor-prefix/README.md

```diff
@@ -9,7 +9,7 @@ a { display: -webkit-flex; }
  *  This prefix */
 ```
 
-This rule ignores non-standard vendor-prefixed values that aren't handled by [Autoprefixer](https://github.com/postcss/autoprefixer).
+This rule does not fix vendor-prefixed values that weren't handled by [Autoprefixer](https://github.com/postcss/autoprefixer) version 10.2.5. Exceptions may be added on a case by case basis.
```

Comment:
For the version check https://github.com/stylelint/stylelint/pull/5312#discussion_r636094886